lib/watch.js

```diff
@@ -15,9 +15,7 @@ module.exports = function (gulp, options) {
   var opts = merge({}, defaults, options);
   var watchers = opts.watchers;
 
-  gulp.task('watch', function () {
-    watchers.forEach(function (item) {
-      gulp.watch(item.match, item.tasks);
-    });
+  gulp.task('watch', () => {
+    watchers.forEach(item => gulp.watch(item.match, item.tasks));
```

Comment:
> Curious about linting/style patterns when it comes to single-line anonymous callbacks.

There is this: http://eslint.org/docs/rules/#ecmascript-6

> **arrow-body-style** - require braces in arrow function body
> **arrow-parens** - require parens in arrow function arguments
> **arrow-spacing** - require space before/after arrow function's arrow (fixable)

I don't think any of these rules address the single-line concern though. IMO, reducing function bodies to a single line is the main benefit of using arrow functions. Perhaps we should come to some rough consensus on when arrow functions should be used (or avoided)?